R/fd_observations.R

```diff
@@ -0,0 +1,116 @@
+#' Number of observations in the FoRTE Dataset
+#'
+#' @note No. of observations for many, but not all datasets
+#'
+#' #' @return A `data.frame` or `tibble` of the observation tally.
+#' Call \code{\link{fd_metadata}} for field metadata.
+#' @export
+#'
+#' @examples
+#' fd_observations()
+fd_observations <- function() {
+
+#
+begin_date <- as.Date("2018-01-01")
+current_date <- Sys.Date()
+
+# create length of dates
+timeframe <- seq.Date(from = begin_date, to = current_date, by = "month", format = "%Y-%m")
+
+# Empty data frame
+timeframe <- data.frame(timeframe)
+
+timeframe$month <- format(timeframe$timeframe, "%m")
+timeframe$year <- format(timeframe$timeframe, "%Y")
+
+#####################
+# CEPTOMETER!
+a <- fd_ceptometer()
+
+# this makes the year and month column
+a$month <- format(as.Date(a$timestamp), "%m")
+a$year <- format(as.Date(a$timestamp), "%Y")
```

Comment:
```suggestion
a$month <- as.numeric(format(as.Date(a$timestamp), "%m"))
a$year <- as.numeric(format(as.Date(a$timestamp), "%Y"))
```

Right? I don't think there's a good reason to use character months/years.